### ISSUE TITLE
Promregator 'Encoded password does not look like BCrypt' 1.0.0-RC6

### ISSUE THREAD

**Issue by braunsonm** [promregator/promregator#230]
## Environment
Promregator 1.0.0-rc6

## Summary / Problem Statement
Since upgrading to Promregator 1.0.0, I am unable to scrape any of the `singleTargetMetrics` with Prometheus. There isn't a stack trace but there is the following error:

```
o.s.s.c.bcrypt.BcryptPasswordEncoder : Encoded password does not look like BCrypt
```

Prometheus sees a 401 response from Promregator.

## Config

```json
{
  "promregator": {
    "authentication": {
      "basic": {
        "username": "admin",
        "password": "MYPASSWORD"
      }
    },
   "authenticator": {
      "type": "NONE"
    },
    "discovery": {
      "auth": "BASIC",
       "hostname": "myhost",
       "port": 1234
    },
    "endpoint": {
       "auth": "BASIC"
    },
    "targets": [
       {"orgName": "myorg", "kubernetesAnnotations": true}
     ]
  },
  "cf": {
    "api_host": "myapi",
    "username": "anotherusername",
    "password": "anotherpassword"
  }
}
```

Downgrading back off the RC fixes the issue.

**Comment by eaglerainbow** [comment]
Thanks for reporting! That's exactly such a case why we need some "beta testers" for the release candidates 😁 
I found the root cause of the problem - and I am working on a fix. Expect a 1.0.0-rc7 for fixing your issue - available soon.

**Comment by eaglerainbow** [comment]
I just released [1.0.0-rc7](https://github.com/promregator/promregator/releases/tag/v1.0.0-rc7).

@braunsonm: Could you please retest, if your issue is gone with that?
Thx!

**Comment by braunsonm** [comment]
Works!
